__tests__/models.test.js

```diff
@@ -0,0 +1,17 @@
+const { selectCommentCountByArticleId } = require('../models/comments-model.js');
+
+const db = require('../db/connection');
+const seed = require('../db/seeds/seed');
+const { topicData, userData, articleData, commentData } = require("../db/data/test-data/index");
```

Comment:
Consider that you are destructuring all of these properties just to put them back into an object when we pass it into the seed 👀